Python/initconfig.c

```diff
@@ -847,7 +847,7 @@ _PyConfig_InitCompatConfig(PyConfig *config)
     config->code_debug_ranges = 1;
     config->cpu_count = -1;
 #ifdef Py_GIL_DISABLED
-    config->enable_gil = _PyConfig_GIL_DEFAULT;
+    config->enable_gil = _PyConfig_GIL_DISABLE;
```

Comment:
My plan had been to leave this line alone and change what `_PyConfig_GIL_DEFAULT` means, [in ceval_gil.c](https://github.com/python/cpython/blob/258408239a4fe8a14919d81b73a16e2cfa374050/Python/ceval_gil.c#L454-L456). That will support the planned behavior of "off by default, but possibly enabled by loading a module." If you make the change here in initconfig.c, the default will be "GIL off and never enabled by module loading" (equivalent to `-X gil=0`).

This distinction only matters once/if gh-116882 lands, but it's worth keeping in mind. I also assigned gh-116329 to you since this PR will close it.

Reply:
Thanks, that makes more sense.